src/main/java/pt/tqs/g205/resources/PratoResource.java

```diff
@@ -0,0 +1,31 @@
+package pt.tqs.g205.resources;
+
+import java.util.List;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RestController;
+import pt.tqs.g205.domain.Prato;
+import pt.tqs.g205.services.PratoService;
+
+/**
+ * 
+ * 
+ *
+ */
+@RestController
+@RequestMapping(value = "/api/pratos")
+public class PratoResource {
+
+  @Autowired
+  private PratoService pratoService;
+
+  @RequestMapping(value = "", method = RequestMethod.GET)
+  public ResponseEntity<?> getAll() {
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove usage of generic wildcard type. ([squid:S1452](https://sonarcloud.io/organizations/bdomingues-github/rules#rule_key=squid%3AS1452))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=pt.tqs.g205%3Arestapi&pullRequest=5&issues=AWNCwXmOXOBV_Yrh-xNv&open=AWNCwXmOXOBV_Yrh-xNv)

[ISSUE_KEY]: AWNCwXmOXOBV_Yrh-xNv